app/code/Magento/Cms/Controller/Adminhtml/Page/Delete.php

```diff
@@ -14,6 +16,23 @@ class Delete extends \Magento\Backend\App\Action
      * @see _isAllowed()
      */
     const ADMIN_RESOURCE = 'Magento_Cms::page_delete';
+    
+    /**
+     * @var \Magento\Cms\Api\PageRepositoryInterface
+     */
+    private $cmsPage;
+    
+    /**
+     * @param Context $context
+     * @param \Magento\Cms\Api\PageRepositoryInterface $cmsPage
+     */
+    public function __construct(
+        Context $context,
+        \Magento\Cms\Api\PageRepositoryInterface $cmsPage
+    ) {
+        parent::__construct($context);
+        $this->cmsPage = $cmsPage ?: \Magento\Framework\App\ObjectManager::getInstance()->get(\Magento\Cms\Api\PageRepositoryInterface::class);
```

Comment:
It's not "cms page", now it's "cms page repository", please rename it

Reply:
Done